Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -0,0 +1,20 @@
+(function initTargetBlank() {
+    function targetBlank() {
+        const links = document.querySelectorAll('a');
+        const currentHostname = window.location.hostname;
+
+        for (let i = 0; i < links.length; i++) {
+            if (!links[i].href.match(/^mailto:/) &&
+                (links[i].hostname !== currentHostname &&
+                (!links[i].href.match(/^javascript:/i)))) {
```

Comment:
This has been merged now, but there are superfluous parentheses here:

```suggestion
                links[i].hostname !== currentHostname &&
                !links[i].href.match(/^javascript:/i)) {
```